base_tier_validation/models/tier_review.py

```diff
@@ -91,4 +95,12 @@ def _compute_todo_by(self):
             rec.todo_by = todo_by
 
     def _get_reviewers(self):
-        return self.reviewer_id + self.reviewer_group_id.users
+        if self.reviewer_id or self.reviewer_group_id.users:
+            return self.reviewer_id + self.reviewer_group_id.users
+        reviewer_field = self.env["res.users"]
+        if self.reviewer_field_id:
+            resource = self.env[self.model].browse(self.res_id)
+            reviewer_field = getattr(resource, self.reviewer_field_id.name, False)
+        if not reviewer_field._name == "res.users":
```

Comment:
You should also check that reviewer_field has a value. Otherwise the review is initiated with no user.

Reply:
This opens the possibility that you set a field that can be empty, if it is empty no review is needed. If you want to ensure a reviewer always make the "reviewer" field required in the reviewed model.